clang/lib/Format/Format.cpp

```diff
@@ -777,7 +779,7 @@ template <> struct MappingTraits<FormatStyle::SpacesInLineComment> {
     IO.mapOptional("Maximum", signedMaximum);
     Space.Maximum = static_cast<unsigned>(signedMaximum);
 
-    if (Space.Maximum != -1u)
+    if (Space.Maximum != std::numeric_limits<unsigned>::max())
```

Comment:
```suggestion
    if (Space.Maximum < UINT_MAX)
```
as that's what's used elsewhere in the file.